Model/ConfigProvider.php

```diff
@@ -187,7 +187,7 @@ public function getActivePaymentMethod()
                 foreach ($collection as $item) {
                     $data             = $item->getData();
                     $id               = $data['id'];
-                    $maskedPan        = $data['masked_pan'] . ' (' . $data['expires'] . ')';
+                    $maskedPan        = $data['masked_pan']; // Last four digits

```

Comment:
Why is it called $maskedPan if this is instead the last four digits?

Reply:
@emicha, I am saving the value in masked_pan by adding the asterisk before the last four digits e.g ********1234. Which is why I didn't rename it.

Reply:
is removal of expiration date from view expected?